From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Wed, 11 Dec 2019 23:12:29 -0800
Subject: [PATCH] Update the generated file list for not-ffmpeg

To regenerate:
  cd src/3rdparty/chromium/third_party/ffmpeg
  perl -pi -e 'if (/^\s*"(lib.*)",/) { $_ = "" unless -f $1; }' ffmpeg_generated.gni
---
 .../third_party/ffmpeg/ffmpeg_generated.gni   | 78 -------------------
 1 file changed, 78 deletions(-)

diff --git a/src/3rdparty/chromium/third_party/ffmpeg/ffmpeg_generated.gni b/src/3rdparty/chromium/third_party/ffmpeg/ffmpeg_generated.gni
index fa2b74e09..5bd90ed33 100644
--- a/src/3rdparty/chromium/third_party/ffmpeg/ffmpeg_generated.gni
+++ b/src/3rdparty/chromium/third_party/ffmpeg/ffmpeg_generated.gni
@@ -42,7 +42,6 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (is_android && curre
     "libavcodec/dirac.c",
     "libavcodec/dv_profile.c",
     "libavcodec/encode.c",
-    "libavcodec/fft_fixed.c",
     "libavcodec/fft_fixed_32.c",
     "libavcodec/fft_float.c",
     "libavcodec/fft_init_table.c",
@@ -55,7 +54,6 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (is_android && curre
     "libavcodec/libopus.c",
     "libavcodec/libopusdec.c",
     "libavcodec/mathtables.c",
-    "libavcodec/mdct_fixed.c",
     "libavcodec/mdct_fixed_32.c",
     "libavcodec/mdct_float.c",
     "libavcodec/mediacodec.c",
@@ -98,7 +96,6 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (is_android && curre
     "libavformat/autorename_libavformat_utils.c",
     "libavformat/avio.c",
     "libavformat/aviobuf.c",
-    "libavformat/cutils.c",
     "libavformat/dump.c",
     "libavformat/flac_picture.c",
     "libavformat/flacdec.c",
@@ -186,8 +183,6 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (is_android && curre
     "libavutil/timecode.c",
     "libavutil/twofish.c",
     "libavutil/tx.c",
-    "libavutil/tx_double.c",
-    "libavutil/tx_float.c",
     "libavutil/tx_int32.c",
     "libavutil/utils.c",
   ]
@@ -212,18 +207,11 @@ if ((current_cpu == "x64" && ffmpeg_branding == "Chrome") || (is_android && curr
   ffmpeg_c_sources += [
     "libavcodec/aac_ac3_parser.c",
     "libavcodec/aac_parser.c",
-    "libavcodec/aacps_float.c",
-    "libavcodec/aacpsdsp_float.c",
-    "libavcodec/aacsbr.c",
-    "libavcodec/aactab.c",
     "libavcodec/adts_header.c",
     "libavcodec/autorename_libavcodec_aacdec.c",
     "libavcodec/autorename_libavcodec_mdct15.c",
     "libavcodec/autorename_libavcodec_sbrdsp.c",
-    "libavcodec/cbrt_data.c",
-    "libavcodec/kbdwin.c",
     "libavcodec/sinewin.c",
-    "libavcodec/sinewin_fixed.c",
     "libavformat/aacdec.c",
     "libavformat/apetag.c",
     "libavformat/img2.c",
@@ -234,21 +222,10 @@ if ((is_mac && ffmpeg_branding == "Chrome") || (is_win && ffmpeg_branding == "Ch
   ffmpeg_c_sources += [
     "libavcodec/cabac.c",
     "libavcodec/h2645_parse.c",
-    "libavcodec/h264_cabac.c",
-    "libavcodec/h264_cavlc.c",
-    "libavcodec/h264_direct.c",
-    "libavcodec/h264_loopfilter.c",
-    "libavcodec/h264_mb.c",
     "libavcodec/h264_parse.c",
     "libavcodec/h264_parser.c",
-    "libavcodec/h264_picture.c",
     "libavcodec/h264_ps.c",
-    "libavcodec/h264_refs.c",
-    "libavcodec/h264_sei.c",
-    "libavcodec/h264_slice.c",
     "libavcodec/h264chroma.c",
-    "libavcodec/h264data.c",
-    "libavcodec/h264dec.c",
     "libavcodec/h264dsp.c",
     "libavcodec/h264idct.c",
     "libavcodec/h264qpel.c",
@@ -262,7 +239,6 @@ if ((is_android && current_cpu == "x64") || (is_android && current_cpu == "x86")
     "libavcodec/x86/constants.c",
     "libavcodec/x86/dct_init.c",
     "libavcodec/x86/fft_init.c",
-    "libavcodec/x86/flacdsp_init.c",
     "libavcodec/x86/mpegaudiodsp.c",
     "libavutil/x86/autorename_libavutil_x86_cpu.c",
     "libavutil/x86/autorename_libavutil_x86_float_dsp_init.c",
@@ -274,7 +250,6 @@ if ((is_android && current_cpu == "x64") || (is_android && current_cpu == "x86")
 
 if ((is_android && current_cpu == "arm" && arm_use_neon) || (use_linux_config && current_cpu == "arm" && arm_use_neon) || (use_linux_config && current_cpu == "arm")) {
   ffmpeg_c_sources += [
-    "libavcodec/arm/fft_fixed_init_arm.c",
     "libavcodec/arm/fft_init_arm.c",
     "libavcodec/arm/flacdsp_init_arm.c",
     "libavcodec/arm/mpegaudiodsp_init_arm.c",
@@ -295,10 +270,7 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (use_linux_config &&
 
 if ((is_android && current_cpu == "x64") || (is_mac) || (is_win && current_cpu == "x64") || (is_win && current_cpu == "x86") || (use_linux_config && current_cpu == "x64") || (use_linux_config && current_cpu == "x86")) {
   ffmpeg_asm_sources += [
-    "libavcodec/x86/dct32.asm",
     "libavcodec/x86/fft.asm",
-    "libavcodec/x86/flacdsp.asm",
-    "libavcodec/x86/imdct36.asm",
     "libavcodec/x86/vorbisdsp.asm",
     "libavutil/x86/cpuid.asm",
     "libavutil/x86/fixed_dsp.asm",
@@ -311,20 +283,13 @@ if ((is_android && current_cpu == "x64") || (is_mac) || (is_win && current_cpu =
 if ((is_mac) || (is_win && current_cpu == "x64") || (is_win && current_cpu == "x86") || (use_linux_config && current_cpu == "x64") || (use_linux_config && current_cpu == "x86")) {
   ffmpeg_c_sources += [
     "libavcodec/x86/autorename_libavcodec_x86_videodsp_init.c",
-    "libavcodec/x86/h264_intrapred_init.c",
     "libavcodec/x86/hpeldsp_init.c",
-    "libavcodec/x86/hpeldsp_vp3_init.c",
-    "libavcodec/x86/vp3dsp_init.c",
     "libavcodec/x86/vp8dsp_init.c",
   ]
   ffmpeg_asm_sources += [
     "libavcodec/x86/autorename_libavcodec_x86_videodsp.asm",
     "libavcodec/x86/fpel.asm",
-    "libavcodec/x86/h264_intrapred.asm",
-    "libavcodec/x86/h264_intrapred_10bit.asm",
     "libavcodec/x86/hpeldsp.asm",
-    "libavcodec/x86/hpeldsp_vp3.asm",
-    "libavcodec/x86/vp3dsp.asm",
     "libavcodec/x86/vp8dsp.asm",
     "libavcodec/x86/vp8dsp_loopfilter.asm",
   ]
@@ -332,9 +297,7 @@ if ((is_mac) || (is_win && current_cpu == "x64") || (is_win && current_cpu == "x
 
 if ((current_cpu == "x64" && ffmpeg_branding == "Chrome") || (is_android && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (is_win && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x64" && ffmpeg_branding == "ChromeOS") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "ChromeOS")) {
   ffmpeg_c_sources += [
-    "libavcodec/x86/aacpsdsp_init.c",
     "libavcodec/x86/mdct15_init.c",
-    "libavcodec/x86/sbrdsp_init.c",
   ]
 }
 
@@ -346,9 +309,7 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (is_android && curre
 
 if ((current_cpu == "x64" && ffmpeg_branding == "Chrome") || (is_win && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x64" && ffmpeg_branding == "ChromeOS") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "ChromeOS")) {
   ffmpeg_asm_sources += [
-    "libavcodec/x86/aacpsdsp.asm",
     "libavcodec/x86/mdct15.asm",
-    "libavcodec/x86/sbrdsp.asm",
   ]
 }
 
@@ -368,26 +329,16 @@ if (use_linux_config && ffmpeg_branding == "ChromeOS") {
     "libavcodec/autorename_libavcodec_qpeldsp.c",
     "libavcodec/autorename_libavcodec_simple_idct.c",
     "libavcodec/autorename_libavcodec_xvididct.c",
-    "libavcodec/celp_filters.c",
     "libavcodec/celp_math.c",
     "libavcodec/error_resilience.c",
     "libavcodec/exif.c",
     "libavcodec/fdctdsp.c",
-    "libavcodec/flvdec.c",
     "libavcodec/gsm_parser.c",
-    "libavcodec/gsmdec.c",
-    "libavcodec/gsmdec_data.c",
-    "libavcodec/h263.c",
     "libavcodec/h263_parser.c",
-    "libavcodec/h263data.c",
-    "libavcodec/h263dec.c",
     "libavcodec/h263dsp.c",
-    "libavcodec/intelh263dec.c",
-    "libavcodec/ituh263dec.c",
     "libavcodec/jfdctfst.c",
     "libavcodec/jfdctint.c",
     "libavcodec/jrevdct.c",
-    "libavcodec/lsp.c",
     "libavcodec/mpeg4video.c",
     "libavcodec/mpeg4video_parser.c",
     "libavcodec/mpeg4videodec.c",
@@ -396,7 +347,6 @@ if (use_linux_config && ffmpeg_branding == "ChromeOS") {
     "libavcodec/mpegutils.c",
     "libavcodec/mpegvideo_motion.c",
     "libavcodec/mpegvideodata.c",
-    "libavcodec/msgsmdec.c",
     "libavcodec/rl.c",
     "libavcodec/tiff_common.c",
     "libavformat/amr.c",
@@ -490,9 +440,7 @@ if ((is_android && current_cpu == "arm" && arm_use_neon) || (use_linux_config &&
     "libavutil/arm/float_dsp_init_neon.c",
   ]
   ffmpeg_gas_sources += [
-    "libavcodec/arm/fft_fixed_neon.S",
     "libavcodec/arm/fft_neon.S",
-    "libavcodec/arm/mdct_fixed_neon.S",
     "libavcodec/arm/mdct_neon.S",
     "libavcodec/arm/rdft_neon.S",
     "libavcodec/arm/vorbisdsp_neon.S",
@@ -513,22 +461,8 @@ if ((use_linux_config && current_cpu == "arm" && arm_use_neon && ffmpeg_branding
 
 if ((is_mac && ffmpeg_branding == "Chrome") || (is_win && current_cpu == "x64" && ffmpeg_branding == "Chrome") || (is_win && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x64" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x64" && ffmpeg_branding == "ChromeOS") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "Chrome") || (use_linux_config && current_cpu == "x86" && ffmpeg_branding == "ChromeOS")) {
   ffmpeg_c_sources += [
-    "libavcodec/x86/h264_qpel.c",
-    "libavcodec/x86/h264chroma_init.c",
-    "libavcodec/x86/h264dsp_init.c",
   ]
   ffmpeg_asm_sources += [
-    "libavcodec/x86/h264_chromamc.asm",
-    "libavcodec/x86/h264_chromamc_10bit.asm",
-    "libavcodec/x86/h264_deblock.asm",
-    "libavcodec/x86/h264_deblock_10bit.asm",
-    "libavcodec/x86/h264_idct.asm",
-    "libavcodec/x86/h264_idct_10bit.asm",
-    "libavcodec/x86/h264_qpel_10bit.asm",
-    "libavcodec/x86/h264_qpel_8bit.asm",
-    "libavcodec/x86/h264_weight.asm",
-    "libavcodec/x86/h264_weight_10bit.asm",
-    "libavcodec/x86/qpel.asm",
   ]
 }
 
@@ -648,25 +582,13 @@ if ((use_linux_config && current_cpu == "x64" && ffmpeg_branding == "ChromeOS")
     "libavcodec/x86/blockdsp_init.c",
     "libavcodec/x86/fdct.c",
     "libavcodec/x86/fdctdsp_init.c",
-    "libavcodec/x86/h263dsp_init.c",
     "libavcodec/x86/idctdsp_init.c",
-    "libavcodec/x86/me_cmp_init.c",
-    "libavcodec/x86/mpegvideo.c",
-    "libavcodec/x86/mpegvideodsp.c",
-    "libavcodec/x86/pixblockdsp_init.c",
-    "libavcodec/x86/qpeldsp_init.c",
-    "libavcodec/x86/xvididct_init.c",
   ]
   ffmpeg_asm_sources += [
     "libavcodec/x86/blockdsp.asm",
-    "libavcodec/x86/h263_loopfilter.asm",
     "libavcodec/x86/idctdsp.asm",
-    "libavcodec/x86/me_cmp.asm",
-    "libavcodec/x86/pixblockdsp.asm",
-    "libavcodec/x86/qpeldsp.asm",
     "libavcodec/x86/simple_idct.asm",
     "libavcodec/x86/simple_idct10.asm",
-    "libavcodec/x86/xvididct.asm",
   ]
 }
 
